core/controllers/base_test.py

```diff
@@ -526,6 +527,17 @@ def test_redirect_oppia_test_server(self):
         self.assertEqual(
             response.headers['Location'], 'https://oppiatestserver.appspot.com')
 
+    def test_splash_redirect(self):
+        # Tests that the old '/splash' URL is redirected to '/'.
+
+        response = self.get_html_response('/splash', expected_status_int=302)
+        response_path = response.headers['location']
+        # The following asserts ensure that the the URL ends with `/` and the
+        # number of backslashes is 3. The structure should be
+        # `http://localhost/`.
+        self.assertEqual(response_path[-1], '/')
```

Comment:
Why can't we test the whole URL? (Instead of this complicated numbers?)

Reply:
Also, using `ends_with` will be more readable here.

Reply:
Done.